From f90cacf71bc5dbf6f5b2bd235de9f99993436500 Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Wed, 31 Aug 2016 17:24:44 +0100
Subject: [PATCH 1/1] Entryfile

---
 entryfile.php | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)
 create mode 100644 entryfile.php

diff --git a/entryfile.php b/entryfile.php
new file mode 100644
index 0000000000..8735a570e9
--- /dev/null
+++ b/entryfile.php
@@ -0,0 +1,19 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+require_once(__DIR__ . '/config.php');
+
+echo 'This file is ok';
-- 
2.9.0

