From 7999a12d1839d17ab9ddaae20cf672ead09bd8bf Mon Sep 17 00:00:00 2001
From: "Eloy Lafuente (stronk7)" <stronk7@moodle.org>
Date: Thu, 29 Sep 2022 09:40:38 +0200
Subject: [PATCH] NOBUG fixture: unbuilt js change for grunt

---
 lib/amd/src/url.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/amd/src/url.js b/lib/amd/src/url.js
index 61dd56b85e8..0150c1dbfa4 100644
--- a/lib/amd/src/url.js
+++ b/lib/amd/src/url.js
@@ -38,7 +38,7 @@ define(['jquery', 'core/config'], function($, config) {
          */
         fileUrl: function(relativeScript, slashArg) {
 
-            var url = config.wwwroot + relativeScript;
+            var url = config.wwwroot + relativeScript + 'hey!';
 
             // Force a /
             if (slashArg.charAt(0) != '/') {
-- 
2.37.2

