From b11edea305738091fdd5a4468585d7f87dc69185 Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Sat, 30 Jul 2016 20:46:05 +0100
Subject: [PATCH 1/1] no return statement

---
 mod/quiz/db/upgrade.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mod/quiz/db/upgrade.php b/mod/quiz/db/upgrade.php
index f42c007..42765c2 100644
--- a/mod/quiz/db/upgrade.php
+++ b/mod/quiz/db/upgrade.php
@@ -193,5 +193,4 @@ function xmldb_quiz_upgrade($oldversion) {
     // Moodle v3.1.0 release upgrade line.
     // Put any upgrade step following this.
 
-    return true;
 }
-- 
2.9.0

