From 2316d2cae62f8f8e63fe4be5bebdc8a15f5045d5 Mon Sep 17 00:00:00 2001
From: "Eloy Lafuente (stronk7)" <stronk7@moodle.org>
Date: Wed, 13 Jan 2021 17:21:41 +0100
Subject: [PATCH] Return type hint in xmldb upgrade functions

---
 lib/db/upgrade.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/db/upgrade.php b/lib/db/upgrade.php
index 685de9e34da..73493473b6e 100644
--- a/lib/db/upgrade.php
+++ b/lib/db/upgrade.php
@@ -84,7 +84,7 @@ defined('MOODLE_INTERNAL') || die();
  * @param int $oldversion
  * @return bool always true
  */
-function xmldb_main_upgrade($oldversion) {
+function xmldb_main_upgrade($oldversion): bool {
     global $CFG, $DB;
 
     require_once($CFG->libdir.'/db/upgradelib.php'); // Core Upgrade-related functions.
-- 
2.30.0

