From c8985b50116b5d27018f428b7e0f2e0678096efa Mon Sep 17 00:00:00 2001
From: "Eloy Lafuente (stronk7)" <stronk7@moodle.org>
Date: Tue, 7 Jun 2022 18:09:41 +0200
Subject: [PATCH] Using more than one unit test class in a file is detected

---
 mod/glossary/tests/lib_test.php | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mod/glossary/tests/lib_test.php b/mod/glossary/tests/lib_test.php
index 9d78bed6c3..fb4df1206a 100644
--- a/mod/glossary/tests/lib_test.php
+++ b/mod/glossary/tests/lib_test.php
@@ -29,6 +29,9 @@ global $CFG;
 require_once($CFG->dirroot . '/mod/glossary/lib.php');
 require_once($CFG->dirroot . '/mod/glossary/locallib.php');
 
+class another_test extends \advanced_testcase {
+}
+
 /**
  * Glossary lib testcase.
  *
-- 
2.36.1

